crates/compiler/can/src/def.rs

```diff
@@ -52,6 +55,9 @@ use roc_types::types::MemberImpl;
 use roc_types::types::OptAbleType;
 use roc_types::types::{Alias, Type};
 use std::fmt::Debug;
+use std::fs;
```

Comment:
Not important right now, but access to the filesystem in the should probably be abstracted away so things like virtual filesystems can be easily plugged in the future.

Reply:
That makes sense. We don't have a pattern for this already, do we?

Reply:
I don't think so

Reply:
I made #6664 to track this. There are quite a few usages of fs in different crates, and I think we should come up with a good general solution.